.travis.yml

```diff
@@ -7,8 +7,11 @@ before_install:
  - sudo gem install -N cocoapods --pre
  - sudo gem install -N xcpretty
  - sudo gem install -N xcpretty-travis-formatter
+ - brew install mxcl/made/swift-sh
```

Comment:
Out of curiosity, what is `swift-sh` used for here?

Reply:
I think you are reviewing an old file @regexident 

But for the record I needed it because I have build and run this script that help me find the best platform version.

```swift
#!/usr/bin/swift sh

import Foundation
import SimulatorControl  // @mackoj ~> 0.2.1

func findSimulatorPlatform(_ args: [String]) {
  
  guard args.count == 2 else {
    print("👉 Usage: ./getSimulatorPlatform.swift xcrunSimctlList.json")
    print("Use: xcrun simctl list -j > xcrun_simctl_list.json for generating the file")
    exit(EXIT_FAILURE)
  }
  
  guard let jsonData = FileManager.default.contents(atPath: args[1]) else {
    print("👉 Failure because JSON file is UNREADABLE 💩")
    exit(EXIT_FAILURE)
  }
  
  // Parse Simulator Info
  SimulatorControlManager.instanciate(jsonData)
  if let platform = SimulatorControlManager.getPlatform() {
    // Expecting: platform=iOS Simulator,name=iPhone 11 Pro Max,OS=13.2
    print(platform)
  } else {
    print("👉 Failure 💩")
    exit(EXIT_FAILURE)
  }
}

findSimulatorPlatform(CommandLine.arguments)
```

It is something that I use a lot when I build ci for iOS.

Reply:
Oh that's nice. I'm so gonna steal that! 😄

Reply:
I'm reworking `SimulatorControl` to make it more flexible so this script might not work for long 😉